### PR TITLE
PMM 7 fix ovf build URLs

### PR DESCRIPTION
**Pull request by talhabinrizwan**

### CODE REVIEW

**Review by EvgeniyPatlan — DISMISSED** [review]

**Review by puneet0191 — APPROVED** [review]

File: pmm/pmm2-ovf.groovy [hunk 1]
```diff
@@ -235,6 +236,7 @@ pipeline {
                             sh '''
                                 FILE=$(ls */*/PMM2-Server-EL9*.ova)
                                 NAME=$(basename ${FILE})
+                                SIZE=\$(du -sh */*/PMM2-Server-EL9*.ova)
```
Comment (atymchuk) [inline pmm/pmm2-ovf.groovy:239]: This doesn't seem to be used anywhere

File: pmm/pmm2-ovf.groovy [hunk 2]
```diff
@@ -252,7 +253,7 @@ pipeline {
                             '''
                         }
                         script {
-                            env.PMM2_SERVER_OVA_S3 = "https://percona-vm.s3.amazonaws.com/PMM2-Server-dev-latest.ova"
+                            env.PMM2_SERVER_OVA_S3 = "http://percona-vm.s3-website-us-east-1.amazonaws.com/PMM2-Server-dev-latest.ova"
```
Comment (atymchuk) [inline pmm/pmm2-ovf.groovy:256]: Http(?s)

**Review by atymchuk — APPROVED** [review]

File: pmm/pmm2-ovf.groovy (hunk 1 above)
Comment (talhabinrizwan) [reply pmm/pmm2-ovf.groovy:239]: Yeah, that was for debugging, will remove it.

File: pmm/pmm2-ovf.groovy [hunk 3]
```diff
@@ -225,7 +226,7 @@ pipeline {
                             """
                         }
                         script {
-                            env.PMM2_SERVER_EL7_OVA_S3 = "https://percona-vm.s3.amazonaws.com/PMM2-Server-dev-latest.el7.ova"
+                            env.PMM2_SERVER_EL7_OVA_S3 = "http://percona-vm.s3-website-us-east-1.amazonaws.com/PMM2-Server-dev-latest.el7.ova"
```
[the remark this answers is not in the document]
Comment (atymchuk) [reply pmm/pmm2-ovf.groovy:228]: Don't know, it seems like we should we reconfigure for security sake :)

[the remark this answers is not in the document]
Comment (EvgeniyPatlan) [reply pmm/pmm2-ovf.groovy:228]: Well this is the link for our internal downloads for tests. It is not the link that will be provided to customers
So for now in order to speedup release process I assume we can proceed. but @talhabinrizwan please create separate jira ticket to improve this

[the remark this answers is not in the document]
Comment (talhabinrizwan) [reply pmm/pmm2-ovf.groovy:228]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteEndpoints.html

**_Important_**
Amazon S3 website endpoints do not support HTTPS or access points. If you want to use HTTPS, you can use Amazon CloudFront to serve a static website hosted on Amazon S3. For more information, see How do I use CloudFront to serve HTTPS requests for my Amazon S3 bucket? To use HTTPS with a custom domain, see Configuring a static website using a custom domain registered with Route 53.

Requester Pays buckets do not allow access through a website endpoint. Any request to such a bucket receives a 403 Access Denied response. For more information, see Using Requester Pays buckets for storage transfers and usage.

**Review by atymchuk — APPROVED** [review]

**Pull request merged**